purchases/src/main/kotlin/com/revenuecat/purchases/UpgradeInfo.kt

```diff
@@ -7,8 +7,12 @@ import com.android.billingclient.api.BillingFlowParams
  * @property oldProductId The old product ID to upgrade from.
  * @property prorationMode The [BillingFlowParams.ProrationMode] to use when upgrading the given oldSku.
  */
-data class UpgradeInfo(
-    // TODO deprecate oldSku
-    val oldSku: String, // TODOBC5 rename?
+data class UpgradeInfo @JvmOverloads constructor(
+    val oldProductId: String,
+    @Deprecated(
+        "Replaced with oldProductId",
+        ReplaceWith("oldProductId")
+    )
+    val oldSku: String = oldProductId,
```

Comment:
Is anyone likely to have specified the name of this field? I'm wondering whether it'd make sense to just rename it. In the end it's exactly the same thing still...

Reply:
yeahhh that's a good point, it's unlikely people were accessing this. i think we've opted for deprecation as a rule, but probably overkill in this scenario